recipe/meta.yaml

```diff
@@ -1,52 +1,58 @@
-{% set version = "2.1.0" %}
+{% set version = "2.3.0" %}
 
 package:
   name: pyviz_comms
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyviz_comms/pyviz_comms-{{ version }}.tar.gz
-  sha256: f4a7126f318fb6b964fef3f92fa55bc46b9218f62a8464a8b18e968b3087dbc0
+  sha256: 44709b2e639e84eb1ae7be5eaf3e28a9be7932d29a8cefce1cd603cb3904ad9c
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - param
     - pip
-    - jupyter-packaging >=0.7.9
-    - jupyterlab >=3.0
+    - jupyter-packaging >=0.7.9,<0.8
+    - jupyterlab >=3.0,<4
     - notebook
     - setuptools
```

Comment:
```suggestion
    - setuptools >=40.8.0
```

Reply:
Due to https://github.com/holoviz/pyviz_comms/blob/abc8f4dfc06c3981aeb1749af91b506dd3e095fc/pyproject.toml#L4